packages/react/src/components/koros/Koros.stories.tsx

```diff
@@ -34,24 +34,73 @@ export const Storm = () => <Koros type="storm" />;
 
 export const Calm = () => <Koros type="calm" />;
 
-export const Flipped = () => (
+export const Flipped = (args) => (
   <>
-    <Koros flipHorizontal />
+    <Koros flipHorizontal={args.flipHorizontal} />
     <br />
     <br />
-    <Koros type="beat" flipHorizontal />
+    <Koros type="beat" flipHorizontal={args.flipHorizontal} />
     <br />
     <br />
-    <Koros type="pulse" flipHorizontal />
+    <Koros type="pulse" flipHorizontal={args.flipHorizontal} />
     <br />
     <br />
-    <Koros type="wave" flipHorizontal />
+    <Koros type="wave" flipHorizontal={args.flipHorizontal} />
     <br />
     <br />
-    <Koros type="storm" flipHorizontal />
+    <Koros type="storm" flipHorizontal={args.flipHorizontal} />
   </>
 );
 
+Flipped.args = {
+  flipHorizontal: true,
+};
+
+export const Rotated = (args) => <Koros type={args.type} flipHorizontal={args.flipHorizontal} rotate={args.rotate} />;
```

Comment:
I would probably either remove, rename or change either the Rotated or the RotatedKorosExample.They are very similar and it can be confusing why there are two examples of the same thing. 

RotatedKorosExample looks very good and it is a practical example of using Koros in a good way. Too bad it does not demo the possibility to rotate the koro shape with the property. 

Maybe renaming here would be the best solution? For example, Rotated and Rotated in a container or something?

Reply:
- [x] rename koros story